src/main/java/org/systemli/keycloak/UserliUserClient.java

```diff
@@ -0,0 +1,34 @@
+package org.systemli.keycloak;
```

Comment:
This interface only makes sense when we would expose an API in Keycloak, which we don't do. So this makes no sense here.